.pre-commit-config.yaml

```diff
@@ -7,15 +7,13 @@ repos:
     rev: 21.5b1
     hooks:
     - id: black
-      language_version: python3.8
+      language_version: python
```

Comment:
Can you explain this and the change on `pyproject.toml` a bit more?
Would there be a way to specify a minimum version? i.e. 3.8+

Reply:
specifying python 3.8 here excludes python 3.9 (resulting in error if that is the runtime python version), I'm not sure that we need to enforce a python version here, leaving it to `setup.py` or README